### PR TITLE
[RU] Ranking Criteria

### PR DESCRIPTION
**Pull request by AnnSur**
:thinking:
Fully completed translation of Ranking Criteria      uwu

### CODE REVIEW

**Comment by cl8n** [comment]
may want to add the changes from https://github.com/ppy/osu-wiki/pull/2160 and https://github.com/ppy/osu-wiki/pull/2156 as well since these will likely not change much before being merged (2160 has precedence over 2156 if they conflict)

**Comment by AnnSur** [comment]
> may want to add the changes from #2160 and #2156 as well since these will likely not change much before being merged (2160 has precedence over 2156 if they conflict)

thanks for helping with that

File: wiki/Ranking_Criteria/ru.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
+Критерии ранкинга
```
Comment (WalterToro) [inline wiki/Ranking_Criteria/ru.md]: Remove the line below and change this to:

```suggestion
# Критерии ранкинга
```

File: wiki/Ranking_Criteria/ru.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+=================
+
+*Для критериев ранкинга определенного режима: [osu!](/wiki/rc_osu!), [osu!taiko](/wiki/rc_osu!taiko), [osu!catch](/wiki/rc_osu!catch), и [osu!mania](/wiki/rc_osu!mania)*
+
+К сведению, к данным критериям относятся последние изменения, обсужденные на [форуме для обсуждения](https://osu.ppy.sh/forum/87). Любое правило, которое обсуждается на данный момент, приводится здесь в действующем виде и будет обновлено после того, как обсуждение завершится.
+
+**Имейте в виду, что [Нормы поведения](https://osu.ppy.sh/wiki/Code_of_Conduct), [руководство по Таймингу песен с #/8-тактом](https://osu.ppy.sh/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures), а также [Правила для содержаний песен](https://osu.ppy.sh/wiki/Ranking_Criteria/Song_Content_Rules) являются частью Критериев Ранкинга и применяются ко всем существующим игровым режимам в osu!.**
+
+
+Глоссарий
```
Comment (WalterToro) [inline wiki/Ranking_Criteria/ru.md]: Revove the line below and change this to:

```suggestion
## Глоссарий
```

File: wiki/Ranking_Criteria/ru.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+
+- **Наивысшая сложность мапсета должна соотноситься с атмосферой песни.** Easy/Normal сложности могут быть использованы как единственная сложность, если их ритмы не переупрощены. В ином случае должна быть добавлена сложность Hard или выше.
+- **Избегайте непонятных комбинаций никнеймов, чтобы указать авторов совместной гостевой сложности.** Если неясно, чьи имена пользователей объединены, рекомендуется упростить это.
+- **Избегайте названий сложностей с образными выражениями, явно не связанных с гостем или уровнем сложности.** (например, Mapper's Tragic Love Extra)
+- **Имена пользователей, указывающие на автора гостевой мапы, должны быть согласованы между несколькими мапсетами.** Изменение ника пользователя затрудняет понимание того, кто создал сложность, и вводит людей в заблуждение.
+- **Избегайте не-алфавитно-цифровых символов Юникода с именем сложности.** Они могут вызывать ошибки в BSS (системе публикации карт) и проблемы для определенных пользователей при появлении в чате.
+
+## Метадата
+
+### Правила
```
Comment (WalterToro) [inline wiki/Ranking_Criteria/ru.md]: New line after this

**Pull request merged**